Apromore-Core-Components/Apromore-BPMNEditor/src/main/scripts/plugins/pdf.js

```diff
@@ -108,26 +108,33 @@ Apromore.Plugins.File = Clazz.extend({
         }
 
         // Send the svg to the server.
-        new Ajax.Request(Apromore.CONFIG.PDF_EXPORT_URL, {
-            method: 'POST',
-            parameters: {
-                resource: resource,
-                data: svgClone,
-                format: "pdf"
-            },
-            onSuccess: (function(request){
-                myMask.hide();
-                // Because the pdf may be opened in the same window as the
-                // process, yet the process may not have been saved, we're
-                // opening every other representation in a new window.
-                // location.href = request.responseText
-                window.open(request.responseText);
-            }).bind(this),
-            onFailure: (function(){
+        var xhr = new XMLHttpRequest();
+        var params = "resource=" + resource + "&data=" + svgClone + "&format=pdf";
+
+        xhr.open("POST", Apromore.CONFIG.PDF_EXPORT_URL);
+        xhr.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
+        xhr.responseType = 'arraybuffer';

```

Comment:
Probably try "blob" directly as responseType, since it's tiny-bit more supported https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseType